cmd/scheduler.go

```diff
@@ -21,10 +23,22 @@ type SchedulerCmdOpt struct {
 }
 
 func getSchedulerCmdOpt() SchedulerCmdOpt {
+	schedulerInterval := viper.GetString("scheduler.interval")
+	if schedulerInterval == "" {
+		schedulerInterval = os.Getenv("SCHEDULER_INTERVAL")
```

Comment:
I think we should still keep getting it with viper, what you need to do is when you bind the PFlag, you can get it from CLI/config/env

Reply:
@realstraw sure, can do that, let me change